composer.json

```diff
@@ -42,6 +42,7 @@
         "drupal/migrate_plus": "^5.1",
         "drupal/migrate_source_csv": "^3.4",
         "drupal/migrate_tools": "^5.0",
+        "drupal/og": "^1.0@alpha",
```

Comment:
I don't think you've export the new content type

Reply:
Do you mean using drush cex command?

Reply:
In whatever way you think is the right way :)

Reply:
added all config file related to OG and new content type in last commit.